### ISSUE TITLE
Links do not work when tap initialized

### ISSUE THREAD

**Issue by ezhlobo** [pukhalski/tap#36]
Links do not work when tap initialized. 

```js
emulatedTap: function( e ) {
  if ( coords.offset ) {
    utils.fireFakeEvent( e, Tap.options.eventName );
  }

  return e.preventDefault();
}
```

**Comment by pukhalski** [comment]
Any test case? I was trying to reproduce but all the links work fine for me together with Tap.

**Comment by pukhalski** [comment]
@ezhlobo, any updates, mate?

**Comment by jhiggins-thrillist** [comment]
It looks like this was introduced with this comment: https://github.com/pukhalski/tap/commit/31dffb40ce3e95396fcd1067d0dcad152aa7a736

You can see an example here: http://codepen.io/j0wy/pen/OVEMjR

For the time being, you can use: https://raw.githubusercontent.com/pukhalski/tap/b5b767cc8c31dade0835d820e3530dd1ebdc8837/dist/tap.min.js

**Comment by qikkeronline** [comment]
@pukhalski I also ran into this issue today - reverted back to the version @jhiggins-thrillist suggested and it works again. Exactly the same behaviour; on desktops no issues but on all ios devices none of the <a> elements function anymore. :+1:

**Comment by pukhalski** [comment]
Funny! Thanks, guys. It seems that we have to find out another way to handle VoiceOver support...

**Comment by JamesJosephFinn** [comment]
I have replicated this issue. I am a copy-pasta man when it comes to js and mostly lean on my teammates, but here is my code:
```

**Comment by JamesJosephFinn** [comment]
I have replicated this issue in my current project. I am a copy-pasta man when it comes to js, and mostly rely on my teammates, but here is my code:
```
document.querySelector('.js-slide-nav-toggle').addEventListener('tap', function (e) {
    var slideNav = document.querySelector('.js-slide-nav');
    var overlay = document.querySelector('.js-slide-nav__overlay');
    classie.add(slideNav, 'js-slide-nav--open');
    classie.remove(slideNav, 'js-slide-nav--closed');
    classie.add(overlay, 'js-slide-nav__overlay--shown');
    classie.remove(overlay, 'js-slide-nav__overlay--hidden');
});
document.querySelector('.js-slide-nav__overlay').addEventListener('tap', function (e) {
    var slideNav = document.querySelector('.js-slide-nav');
    classie.add(slideNav, 'js-slide-nav--closed');
    classie.remove(slideNav, 'js-slide-nav--open');
    classie.add(this, 'js-slide-nav__overlay--hidden');
    classie.remove(this, 'js-slide-nav__overlay--shown');
});
```
@jhiggins-thrillist patch fixes it. Thanks so much!

**Comment by pukhalski** [comment]
Thanks, gentlemen, for all your responses! I'm really happy to see that the lib is being used.

But some of bad news to come: I still haven't found a fix for this links issue, so I am going to remove VoiceOver support fix in order to close this a bit more major issue. Thanks.

**Comment by ezhlobo** [comment]
Fixed in new version 1.0.8. Thanks!
